ts/Data/Connectors/DataConnector.ts

```diff
@@ -100,6 +100,12 @@ abstract class DataConnector implements DataEvent.Emitter {
      */
     public readonly table: DataTable;
 
+    /**
+     * The corresponding request abort controller, which gets assigned when data
+     * polling starts. Used to abort the request when data polling stops.
+     */
+    public abortController?: AbortController;
```

Comment:
1. I would suggest replacing it with `pollingController`, which is much more meaningful. Especially, that in the further part of code we use `abortController.abort();`
2. The doclet is a bit not understandable.

Reply:
I agree with the naming, and the changes have been pushed. Thanks!